.changeset/quick-rings-wave.md

```diff
@@ -0,0 +1,7 @@
+---
+"@rocket.chat/meteor": patch
+"@rocket.chat/i18n": patch
+"@rocket.chat/ui-client": patch
```

Comment:
Should be minor imho

Reply:
Its a new feature in the workspace settings, do we usually add things like this as a minor?

Reply:
Yep